vcap/vcap/capsule.py

```diff
@@ -102,8 +102,7 @@ def process_frame(self,
             # Note the inplace shuffle, but this should be okay, esp. w/ lock
             random.shuffle(self.backends)
             laziest_backend: BaseBackend \
-                = min(self.backends,
-                      key=lambda backend: backend.oven.total_imgs_in_pipeline)
+                = min(self.backends, key=lambda backend: backend.workload)
```

Comment:
Github won't let me add a comment to the line I want to... but you forgot to change the comment that refers to `BaseBackend.oven.total_imgs_in_pipeline`

Reply:
Awesome, good catch!